doc/guide/https.xml

```diff
@@ -44,6 +44,9 @@
       platforms, Cockpit will also generate a ca.crt in that directory, which
       may be safely imported into client browsers.</para>
 
+    <para>The permissions and ownership of the files are not important: they will be
+      accessed as root.</para>
```

Comment:
"not important" is maybe too lax -- they are still important in the sense that the key must be protected. Maybe "Cockpit will read the files as root, so they can have tight permissions"?

Reply:
Took your suggestion here, thanks.